test/dataset_collection/test_dataset_collection.py

```diff
@@ -329,6 +344,21 @@ def test_inline_strange_body2(self):
                 open(test_filepath, encoding='utf-8') as test_ex:
             self.assertEqual(actual_file.read(), test_ex.read())
 
+    def test_inline_comments_at_the_end(self):
+        filepath = self.current_directory / 'InlineExamples' / 'ObjectProperties_cut.java'
+        test_filepath = self.current_directory / 'InlineTestExamples' / 'ObjectProperties_cut.java'
+        algorithm_type = InlineTypesAlgorithms.WITHOUT_RETURN_WITHOUT_ARGUMENTS
+        algorithm_for_inlining = AlgorithmFactory().create_obj(algorithm_type)
+        ast = AST.build_from_javalang(build_ast(filepath))
+        m_decl = [
+            x for x in ast.get_proxy_nodes(ASTNodeType.METHOD_DECLARATION)
+            if x.name == 'updateSashWidths'][0]
```

Comment:
non intuitive formatting.
```suggestion
            if x.name == 'updateSashWidths'
        ][0]
```